### DISCUSSION TITLE
REMINDER:  VSOMEIP Meeting Monday, November 20th at 7am PT / 10am ET / 16:00 CET

### DISCUSSION THREAD

**Discussion by paulboyes** [COVESA/vsomeip#575] [Announcements]
Just a reminder.  We have our regular monthly meeting Monday.   See [Community Calendar](https://wiki.covesa.global/display/WIK4/COVESA+Common+Meeting+Schedule) for detail.  

[ Zoom Link](https://us06web.zoom.us/j/86228511243?pwd=K2E3cW5XV085Q0NvUkRqV0NXZVFKUT09)

**Comment by jennyj1** [comment]
I have been working on some task and my system requires communication **between two devices** particularly docker containers. The messages has to be sent without service discovery. By this I mean that there is **no multicast address** in our system. Also, another requirement is that the message type has to be of **notification type (0x02).**

1.	 Is this communication feasible with VSOMEIP repo?
2.	If no, why?
3.	If yes, how can I achieve this?
NOTE: There is no implementation for notify and subscribe without service discovery.
NOTE: I have found a requirement in SOME/IP documentation saying --> SOME/IP is only used for transporting the updated value and not for publishing and subscribing mechanisms. These mechanisms are implemented by SOME/IP SD. Hence, is publish subscribe not possible **without SD**?

**Comment by nachtfuchs** [comment]
Thanks for the option to talk with you guys. It was really informative and useful. Please keep up the work you are doing.
